src/components/MobileNavBar.js

```diff
@@ -0,0 +1,217 @@
+import React, { useContext } from 'react';
+import { NavLink } from 'react-router-dom';
+import Modal from 'react-modal';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBars } from '@fortawesome/free-solid-svg-icons';
+import Search from './SearchIcon';
+import { AuthContext } from '../context/AuthContext';
+
+const customStyles = {
+  content: {
+    top: '45%',
+    left: '50%',
+    right: 'auto',
+    bottom: 'auto',
+    marginRight: '-50%',
+    transform: 'translate(-50%, -50%)',
+    width: '80%',
+    background: 'rgb(151 191 15 / 92%)',
+  },
+};
+
+const MobileNavbar = () => {
+  const [modalIsOpen, setIsOpen] = React.useState(false);
+
+  const openModal = () => {
+    setIsOpen(true);
+  };
+
+  // const afterOpenModal = () => {
+  //   // references are now sync'd and can be accessed.
+  //   subtitle.style.color = '#f00';
+  // };
```

Comment:
- [ ] Please, consider deleting this old piece of commented code to improve the cleanliness of the repo. 👍